### PR TITLE
Default json rpc variant to geth

### PR DESCRIPTION
**Pull request by vbaranov**
Related to https://github.com/blockscout/blockscout/pull/10509

### CODE REVIEW

File: for-developers/information-and-settings/env-variables/backend-env-variables.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ _**Note**_**: Before release 5.1.2, all environment variables of time format sup
...
+| `ETHEREUM_JSONRPC_VARIANT`                              | Tells the application which RPC Client the node is using (i.e. `erigon`, `geth`, `nethermind`, `besu`, `filecoin`, or `ganache`) (See Client Settings for more info)                                                                                                                                                                                                                                                                               | <p>Required: ✅<br>Version: All<br>Default: <code>geth</code><br>Applications: API, Indexer</p>                                                                          |
```
Comment (k1rill-fedoseev) [inline for-developers/information-and-settings/env-variables/backend-env-variables.md:21]: Let's update this page too - https://docs.blockscout.com/for-developers/information-and-settings/requirements/client-settings#supported-clients

Comment (vbaranov) [reply for-developers/information-and-settings/env-variables/backend-env-variables.md:21]: Good point. I've actualized that page. Please check in https://github.com/blockscout/docs/pull/305/commits/4f7808c6a6d221cea17be13850ea144b93df99d3.

File: for-developers/information-and-settings/client-settings.md [hunk 2]
```diff
@@ -1,102 +1,93 @@
...
 
-1. `parity` the same for Parity, OpenEthereum and Nethermind (default)
-2. `erigon`
-3. `geth`
-4. `besu`
-5. `ganache`
+| JSON RPC Client | Value          | Note                                                                              |
+| --------------- | -------------- | --------------------------------------------------------------------------------- |
+| Geth\\Reth      | \`geth\`       | Default. This value is applicable for both Geth and Reth JSON RPC clients                  |
+| Erigon          | \`erigon\`     |                                                                                   |
```
Comment (k1rill-fedoseev) [inline for-developers/information-and-settings/client-settings.md]: Let's add a note, that `reth` based instances should use `erigon` variant too (or maybe `nethermind`).

Comment (vbaranov) [reply for-developers/information-and-settings/client-settings.md]: Added here https://github.com/blockscout/docs/pull/305/commits/2d491730ea98a6f523931b149779c84d958c7e32.

**Review by k1rill-fedoseev — APPROVED** [review]

**Pull request merged**